config/initializers/flipper.rb

```diff
@@ -47,6 +47,10 @@
     # default features to enabled for development and test only
     Flipper.enable(feature) if Rails.env.development? || Rails.env.test?
```

Comment:
Alternatively, we could change this line to 
` Flipper.enable(feature) if (Rails.env.development? || Rails.env.test?) && !feature.match(/ssoe/)`
but that wouldn't help local users who already have the toggle set in their local DB.